### DISCUSSION TITLE
bugged contribution?

### DISCUSSION THREAD

**Discussion by p3rc1us** [community/community#120502] [New to GitHub] [answered]
Hello, I would like to know if this is normal. when I just click my profile and scroll down to my contribution chart, this is what it looks like

![image](https://github.com/community/community/assets/144686514/9192d510-b4e2-4375-bb98-d2c84312cefd)

but when i click the 2024 some of the bright greens became dull like this

![image](https://github.com/community/community/assets/144686514/46edb150-c9e4-4af9-bc9b-3fc314fb67e7)

is this a bug?

**Comment by eddysantiagoo** [comment]
Hi @p3rc1us  this is normal let me explain. In the first image you can see data for a natural year (365 days). It shows you the last 365 days without filtering by any specific year, whether 2023 or 2024. However, when you click on the label '2024' or '2023' GitHub will then only display the data for the 365 days of the year selected. Since today is in April of 2024, the second image shows data from January 1, 2024, to today’s date. In simple terms this is totally normal :)

If this answer helps you, please mark it as the correct answer.

**Reply by p3rc1us** [reply, marked as answer]
hello Eddy! I get what you're trying to say. but anyway, my chart seems good now right after I read your response. Notice that there are slight differences in colors, unlike the 2024 from above.  I appreciate your response brother, thanks again.

![image](https://github.com/community/community/assets/144686514/23e44324-e457-4a9a-b582-36aea8f319fb)
